### ISSUE TITLE
Scheduling more than two jobs, starts failing with "job startup timeout"

### ISSUE THREAD

**Issue by koureasstavros** [citusdata/pg_cron#253]
I have Postgres15 with latest version of pg_cron installed.
I have created some stored procedures to update some data, these stored procedures can be executed fine with CALL public.procedure.
Scheduling with some minutes recurring one job to call one procedure is running fine.
Scheduling with some minutes recurring two jobs to call two procedures are running fine.
Scheduling with some minutes recurring three jobs to call three procedures are staring failing.
The error is "job startup timeout"

![image](https://user-images.githubusercontent.com/6669704/226371103-6ee2b1e7-4a50-45ba-8d75-0291b5acadef.png)

Any idea?

**Comment by koureasstavros** [comment]
Channing (increasing) these two parameters in file (/var/lib/postgresql/data/postgresql.conf) seems to fixed my problem, now I need to understand them better:
add max_parallel_workers = 256
add max_worker_processes = 65535

**Comment by DotNetRico** [comment]
Is there any news? I have the same problem

**Comment by koureasstavros** [comment]
Yes, you should change some properties of pg_cron like pg_cron_jobs
